PINCache/PINCache.h

```diff
@@ -223,6 +236,17 @@ typedef void (^PINCacheObjectContainmentBlock)(BOOL containsObject);
 - (void)setObject:(id <NSCoding>)object forKey:(NSString *)key;
 
 /**
+ Stores an object in the cache for the specified key and the specified memory cost. If the cost causes the total
+ to go over the <memoryCache.costLimit> the cache is trimmed (oldest objects first). This method blocks the calling thread
+ until the object has been stored.
+ 
+ @param object An object to store in the cache.
+ @param key A key to associate with the object. This string will be copied.
+ @param cost An amount to add to the <memoryCache.totalCost>.
+ */
+- (void)setObject:(nullable id)object forKey:(nullable NSString *)key withCost:(NSUInteger)cost;
```

Comment:
I think this should be:
`- (void)setObject:(id <NSCoding>)object forKey:(NSString *)key withCost:(NSUInteger)cost;` (id shouldn't be nullable should be conforming to NSCoding and key shouldn't be nullable)